table.csv

```diff
@@ -1,462 +1,462 @@
+bitcoin-tx,                     ipld,           0xb1,           draft,     Bitcoin Tx
+bitcoin-witness-commitment,     ipld,           0xb2,           draft,     Bitcoin Witness Commitment
+zcash-block,                    ipld,           0xc0,           draft,     Zcash Block
+zcash-tx,                       ipld,           0xc1,           draft,     Zcash Tx
+docid,                          namespace,      0xce,           draft,     Ceramic Document Id
+stellar-block,                  ipld,           0xd0,           draft,     Stellar Block
+stellar-tx,                     ipld,           0xd1,           draft,     Stellar Tx
+md4,                            multihash,      0xd4,           draft,
+md5,                            multihash,      0xd5,           draft,
+bmt,                            multihash,      0xd6,           draft,     Binary Merkle Tree Hash
+decred-block,                   ipld,           0xe0,           draft,     Decred Block
+decred-tx,                      ipld,           0xe1,           draft,     Decred Tx
+ipld-ns,                        namespace,      0xe2,           draft,     IPLD path
+ipfs-ns,                        namespace,      0xe3,           draft,     IPFS path
+swarm-ns,                       namespace,      0xe4,           draft,     Swarm path
+ipns-ns,                        namespace,      0xe5,           draft,     IPNS path
+zeronet,                        namespace,      0xe6,           draft,     ZeroNet site address
+secp256k1-pub,                  key,            0xe7,           draft,     Secp256k1 public key
+bls12_381-g1-pub,               key,            0xea,           draft,     BLS12-381 public key in the G1 field
+bls12_381-g2-pub,               key,            0xeb,           draft,     BLS12-381 public key in the G2 field
+x25519-pub,                     key,            0xec,           draft,     Curve25519 public key
+ed25519-pub,                    key,            0xed,           draft,     Ed25519 public key
+bls12_381-g1g2-pub,             key,            0xee,           draft,     BLS12-381 concatenated public keys in both the G1 and G2 fields
+dash-block,                     ipld,           0xf0,           draft,     Dash Block
+dash-tx,                        ipld,           0xf1,           draft,     Dash Tx
+swarm-manifest,                 ipld,           0xfa,           draft,     Swarm Manifest
+swarm-feed,                     ipld,           0xfb,           draft,     Swarm Feed
+udp,                            multiaddr,      0x0111,         draft,
+p2p-webrtc-star,                multiaddr,      0x0113,         draft,
+p2p-webrtc-direct,              multiaddr,      0x0114,         draft,
+p2p-stardust,                   multiaddr,      0x0115,         draft,
+p2p-circuit,                    multiaddr,      0x0122,         permanent,
+dag-json,                       ipld,           0x0129,         draft,     MerkleDAG json
```

Comment:
I would consider dag-json being permanent. But we can also discuss this in a seperate PR.